src/client.rs

```diff
@@ -74,6 +77,165 @@ pub extern "C" fn rustls_client_config_builder_build(
     }
 }
 
+/// Currently just a placeholder with no accessors yet.
+/// https://docs.rs/rustls/0.19.0/rustls/struct.RootCertStore.html
+#[allow(non_camel_case_types)]
+pub struct rustls_root_cert_store {
+    _private: [u8; 0],
+}
+
+/// Input to a custom certificate verifier callback. See
+/// rustls_client_config_builder_dangerous_set_certificate_verifier().
+#[allow(non_camel_case_types)]
+#[repr(C)]
+pub struct rustls_verify_server_cert_params<'a> {
+    end_entity: rustls_slice_bytes<'a>,
+    intermediates: rustls_slice_slice_bytes<'a>,
```

Comment:
I'm assuming `end_entity` and `intermediates` are certificates, but that's not obvious since the types are byte slices. This would be more clear as `end_entity_cert` and `intermediate_certs`.